src/Build.UnitTests/BackEnd/BuildResult_Tests.cs

```diff
@@ -347,6 +350,27 @@ public void TestTranslation()
             Assert.True(TranslationHelpers.CompareCollections(result["omega"].Items, deserializedResult["omega"].Items, TaskItemComparer.Instance));
         }
 
+        [Fact]
+        public void TaskReturnsFailureButDoesNotLogErrorShouldCauseBuildFailure()
+        {
+
+            using (TestEnvironment env = TestEnvironment.Create())
```

Comment:
```suggestion
            using (TestEnvironment env = TestEnvironment.Create(_output))
```

this doesn't usually help much but doesn't hurt either